pkgs/games/starruler2/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  callPackage,
+  makeWrapper,
+  # Path to directory containing music for the game,
+  # such as `data/music` from the commercial version.
+  musicPath ? null,
+  # Paths of mods to add.
+  modPaths ? [],
+}: let
+  data = callPackage ./data.nix {};
+  engine = callPackage ./engine.nix {};
+in
+  # The game cannot access files outside the working directory,
+  # so it cannot access files behind symlinks.
+  stdenv.mkDerivation {
+    pname = "starruler2";
+    version = lib.trivial.max data.version engine.version;
+
+    phases = ["installPhase"];
+
+    nativeBuildInputs = [makeWrapper];
+
+    installPhase = ''
+      game="$out/share/games/starruler2"
+      mkdir -p "$out/bin"
+
+      cp -r "${data}/share" "$out/share"
+      chmod -R u+w "$out/share"
```

Comment:
why the chmod ?

Reply:
I think this was a holdover. I removed it.